crates/errors/src/error_codes/META0010.md

```diff
@@ -6,5 +6,5 @@ Suggestions:
 
 * Up/Downgrade your Restate server to the requested version.
 * Migrate your data to the requested version by running the migration scripts.
-* Wipe your meta storage directory to start afresh via `--wipe=meta`.
+* Wipe your meta storage directory to start afresh via `rm -rf <BASE_DIR>/${HOST}/local-metadata-store`.
```

Comment:
Maybe we replace `${HOST}` with `<NODE_NAME>`. The node name defaults to `${HOST}` but if someone configures the node name, this would be different.